test/lint/lint-python-dead-code-whitelist

```diff
@@ -0,0 +1,93 @@
+_.posix_path  # unused attribute (contrib/macdeploy/custom_dsstore.py:45)
```

Comment:
Please remove comments with line numbers (which are not stable and will not be updated when those files are changed), and deduplicate this list.

Reply:
I think it wastes less time if we just keep the autogenerated file as is

Reply:
I don't understand. Why add comments here that will be wrong almost instantly?

Reply:
Only the line number will be wrong. The rest should still be valid and might even be useful. I don't see a reason to manually fix the line number every time this file is autogenerated.

Reply:
I don't get it. Every time a new function that triggers a false positive in vulture is added, this should be regenerated? That'll create a huge diff. How is that supposed to be reviewed?

In general, I don't think it's good practice to use the output of a tool to generate its own whitelist. The whitelist should be generated (and reviewed) by someone outside the tool declaring that these functions should be ignored.

Reply:
Ok, reviewed and manually fixed up the output of the tool